src/index.html

```diff
@@ -12,7 +12,57 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+
+    <container class="stars__container">
+
+      <article class="stars stars--0">
```

Comment:
Remember to use semantic tags. Instead of a generic `<article>` for stars, consider if a more descriptive tag could be used, such as `<section>` or `<div>`, unless each `article` represents a self-contained composition in a document, page, application, or site, which would be suitable for independent distribution or reuse.